api-2.2/src/main/java/org/openmrs/module/fhir2/api/dao/impl/FhirConditionDaoImpl_2_2.java

```diff
@@ -42,6 +52,47 @@ public Condition getConditionByUuid(String uuid) {
 		        .uniqueResult();
 	}
 	
+	private ConditionClinicalStatus convertStatus(String status) {
+		if ("active".equalsIgnoreCase(status)) {
+			return ConditionClinicalStatus.ACTIVE;
+		}
+		if ("inactive".equalsIgnoreCase(status)) {
+			return ConditionClinicalStatus.INACTIVE;
+		}
+		// TODO during review: What value of the spec. should be mapped to HISTORY_OF?
+		// http://www.hl7.org/fhir/valueset-condition-clinical.html
+		return ConditionClinicalStatus.HISTORY_OF;
```

Comment:
We shouldn't be, since `HISTORY_OF` is not semantically equivalent to any of those values. Normally, I'd say we just add a custom valueset, but that obviously makes us non-conformant. I think we need to handle `HISTORY_OF` via an extension. Ugh...

Reply:
Yeah, that's fine.

I don't know whether it's on its way to being deprecated or not, but I hope so. Maybe I can propose that core adopts the second level values from the HL7 value set, since "remission" and "resolved" seem more semantically meaningful than just "history of".